cmd/core-command/res/configuration.toml

```diff
@@ -54,12 +54,12 @@ Type = "consul"
   AuthMode = "usernamepassword"                   # required for redis messagebus (secure or insecure).
   SecretName = "redisdb"
     [MessageQueue.Internal.Topics]
-    RequestTopicPrefix = "edgex/command/request"    # for publishing requests to the device service; <device-service>/<device-name>/<command-name>/<method> will be added to this publish topic prefix
-    ResponseTopic = "edgex/command/response/#"      # for subscribing to device service responses
-    InternalRequestCommandTopic = "/command/request/#"     # for subscribing to internal command requests
-    InternalResponseCommandTopicPrefix = "/command/response/"    # for publishing responses back to internal service /<device-name>/<command-name>/<method> will be added to this publish topic prefix
-    InternalRequestQueryTopic = "/commandquery/request"   # for subscribing to internal command query requests
-    InternalResponseQueryTopic = "/commandquery/response"   # for publishing reponsses back to internal service
+    DeviceRequestTopicPrefix = "edgex/command/request"    # for publishing requests to the device service; <device-service>/<device-name>/<command-name>/<method> will be added to this publish topic prefix
+    DeviceResponseTopic = "edgex/command/response/#"      # for subscribing to device service responses
+    RequestCommandTopic = "/command/request/#"     # for subscribing to internal command requests
+    ResponseCommandTopicPrefix = "/command/response"     # for publishing responses back to internal service /<device-name>/<command-name>/<method> will be added to this publish topic prefix
+    RequestQueryTopic = "/commandquery/request"   # for subscribing to internal command query requests
+    ResponseQueryTopic = "/commandquery/response"   # for publishing reponsses back to internal service
```

Comment:
according to your naming convention, you should either rename `RequestDeviceTopicPrefix` or `CommnadRequestTopic`